server/models/action-inputs-model.js

```diff
@@ -6,15 +6,34 @@ function getHint (actionId) {
   return actionId === 'FG1' ? 'How long is the fence you want to claim on in metres?' : 'What is the area you wish to claim on in hectares?'
 }
 
-function actionsInputsModel (parcelRef, actionId, error) {
+function boundsValid (action) {
+  return action && action.inputs && action.inputs[0] && action.inputs[0].upperbound && action.inputs[0].lowerbound
```

Comment:
I modified the parameter here to be `input` rather than `inputs`, and to be an object, because it didn't make sense at this point to have an array. Therefore, this will have to change slightly